tests/unit/models/skill-test.js

```diff
@@ -10,3 +10,14 @@ test('it exists', function(assert) {
   // let store = this.store();
   assert.ok(!!model);
 });
+
+test('it has all of its attributes', function(assert) {
+  assert.expect(3);
+
+  const skill = this.subject();
+  const attributes = Object.keys(skill.toJSON());
+
+  assert.ok(attributes.includes('description'), 'should have description attribute');
+  assert.ok(attributes.includes('title'), 'should have title attribute');
+  assert.ok(attributes.includes('matched'), 'should have matched attribute');
```

Comment:
@JoshSmith ya, I realize that too. I figure I'll just make a Pr once that is in to fix up all the already written tests.